subgraphs/minichefV2/mappings/miniChefV2.ts

```diff
@@ -0,0 +1,236 @@
+/* eslint-disable prefer-const */
+import { Address } from "@graphprotocol/graph-ts";
+import { Farm, Minichef } from "../generated/schema";
+import {
+  convertTokenToDecimal,
+  createUser,
+  BI_18,
+  createLiquidityPosition,
+  createFarm,
+  createUpdateFarmRewards,
+  ZERO_BI,
+  createUpdateReWarder,
+  createUpdateMiniChef,
+} from "./helpers";
+import {
+  Deposit,
+  PoolAdded,
+  Withdraw,
+  EmergencyWithdraw,
+  PoolSet,
+  LogRewardPerSecond,
+  LogRewardsExpiration,
+} from "../generated/MiniChefV2/MiniChefV2";
+
+export function handlePoolAdded(event: PoolAdded): void {
+  // log.info("============== handlePoolAdded =======", []);
+  createFarm(
+    event.address,
+    event.params.pid,
+    event.params.lpToken,
+    event.params.rewarder,
+    event.params.allocPoint
+  );
+}
+
+export function handleDeposit(event: Deposit): void {
+  // log.info("============== handleDeposit =======", []);
+
+  let farmKey =
+    event.address.toHexString() + "-" + event.params.pid.toHexString();
+
+  // log.info("===================farmKey:{}", [farmKey]);
+
+  let farm = Farm.load(farmKey);
+
+  // log.info("===================  farm.tvl before plus :{}", [
+  //   farm.tvl.toString(),
+  // ]);
+  let convertedAmount = convertTokenToDecimal(event.params.amount, BI_18);
+
+  farm.tvl = farm.tvl.plus(convertedAmount);
+
+  // log.info("===================  farm.tvl after plus :{}", [
+  //   farm.tvl.toString(),
+  // ]);
+
+  farm.save();
+
+  // user stats
+  createUser(event.params.to);
+
+  if (event.params.user.notEqual(event.params.to)) {
+    let fromUserLiquidityPosition = createLiquidityPosition(
+      farm.pairAddress as Address,
+      event.params.user,
+      farmKey
+    );
+    fromUserLiquidityPosition.stakedTokenBalance = fromUserLiquidityPosition.stakedTokenBalance.minus(
```

Comment:
I'm not sure what `stakedTokenBalance` is supposed to be tracking

if it represents liquidity (PGL) for an address regardless of being staked, it doesn't make sense to change `stakedTokenBalance` when `user == to` on deposit

if it represents liquidity (PGL) for an address that is currently staked in a farm, it doesn't make sense to decrease `stakedTokenBalance` on deposit

Reply:
`stakedTokenBalance` represent PGL that is currently staked in a farm. what should we do if `user !== to` in this case. I believe we should only do `toAmount = toAmount + convertedAmount`. I am little bit confused here